updater/spec/dependabot/integration_spec.rb

```diff
@@ -1,379 +0,0 @@
-          }
-        ],
-        existing_pull_requests: [],
-        ignore_conditions: [],
-        security_advisories: [],
-        package_manager: "bundler",
-        source: {
-          "provider" => "github",
-          "repo" => "dependabot-fixtures/dependabot-test-ruby-package",
-          "directory" => "/",
-          "api-endpoint" => "https://api.github.com/",
-          "hostname" => "github.com",
-          "branch" => nil
-        },
-        credentials: [{
-          "type" => "git_source",
-          "host" => "github.com",
-          "username" => "x-access-token",
-          "password" => "github-token"
-        }],
-        lockfile_only: false,
-        requirements_update_strategy: nil,
-        update_subdependencies: false,
-        updating_a_pull_request: false,
-        vendor_dependencies: false,
-        security_updates_only: false
-      )
-    end
-
-    it "updates dependencies correctly" do
-      expect(api_client).
-        to receive(:create_pull_request) do |id, deps, files, commit_sha|
-          expect(id).to eq(1)
-          dep = Dependabot::Dependency.new(
-            name: "dummy-pkg-b",
-            package_manager: "bundler",
-            version: "1.2.0",
-            previous_version: "1.1.0",
-            requirements: [
-              { requirement: "~> 1.2.0",
-                groups: [:default],
-                source: nil,
-                file: "Gemfile" }
-            ],
-            previous_requirements: [
-              { requirement: "~> 1.1.0",
-                groups: [:default],
-                source: nil,
-                file: "Gemfile" }
-            ]
-          )
-          expect(deps).to eql([dep])
-          expect(files).to eq(
-            [
-              {
-                "name" => "Gemfile",
-                "content" => fixture("bundler/updated/Gemfile"),
-                "directory" => "/",
-                "type" => "file",
-                "mode" => "100644",
-                "support_file" => false,
-                "content_encoding" => "utf-8",
-                "deleted" => false,
-                "operation" => "update"
-              },
-              {
-                "name" => "Gemfile.lock",
-                "content" => fixture("bundler/updated/Gemfile.lock"),
-                "directory" => "/",
-                "type" => "file",
-                "mode" => "100644",
-                "support_file" => false,
-                "content_encoding" => "utf-8",
-                "deleted" => false,
-                "operation" => "update"
-              }
-            ]
-          )
-          expect(commit_sha).to eq("sha")
-        end
-      end_to_end_job.run
-    end
-
-    it "summarizes the changes" do
-      expect(Dependabot.logger).to receive(:info).with(/Changes to Dependabot Pull Requests/) do |log_message|
-        expect(log_message).to include("created", "dummy-pkg-b ( from 1.1.0 to 1.2.0 )")
-      end
-
-      end_to_end_job.run
-    end
```

Comment:
Is this tested elsewhere? I don't think this is checked in the smoke test? If not it would be worth preserving somehow.